### DISCUSSION TITLE
Informacion

### DISCUSSION THREAD

**Discussion by DavidG2115** [Launch-X-Latam/MisionFrontEnd#523] [Team Posts]
Hola, No me ha llegado el correo para unirme al canal de teams y veo que están haciendo equipos, me podrían dar información. :)

**Comment by UlisesNh** [comment]
Que onda, David! dale click a este enlace para unirte 
https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A59Tb8L_0PlM5uxQtC8AjVfcF-wvYCKBxsS1JRWZLnyo1%40thread.tacv2%2Fconversations%3FgroupId%3De6804a94-ce40-41ca-a5c2-b287fc50036b%26tenantId%3D4ae54b05-b77e-4224-aef1-8661422e0816&type=team&deeplinkId=3c6806b7-685d-4d93-9ffc-906dc320f8e3&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true

**Comment by DavidG2115** [comment]
Va muchas gracias
